README.md

```diff
@@ -199,21 +200,21 @@ Alongside the built assets a metadata file will be created with info about the b
 
 The Metadata class is contained in the [Metadata.groovy](https://github.com/adoptium/ci-jenkins-pipelines/blob/master/pipelines/library/src/common/MetaData.groovy) file and the Json is constructed and written in the [openjdk_build_pipeline.groovy](https://github.com/adoptium/ci-jenkins-pipelines/blob/master/pipelines/build/common/openjdk_build_pipeline.groovy) file.
 
-It is worth noting the additional tags on the semver is the adopt build number.
+It is worth noting the additional tags on the SemVer is the adopt build number.
 
 Below are all of the keys contained in the metadata file and some example values that can be present.
 
 ---
 
 - `vendor:`
-Example values: [`Temurin`, `Alibaba`]
+Example values: [`Temurin`, `Alibaba`, `Adoptium`, `Huawei`]
```

Comment:
We should probably remove `Temurin` and `Adoptium` from this list and use `Eclipse Adoptium` as the value since that's what we use in all places as far as I'm aware.

Reply:
thanks! I will make a new commit to fix this